spvc/test/run_spirv_cross_tests.py

```diff
@@ -540,28 +531,54 @@ def main():
         known_failures = f.read().splitlines()
 
     known_failures = set(
-        map(lambda x: (x.split(',')[0], x.split(',')[1] == 'True'), known_failures))
+        map(lambda x: (x.split(',')[0], x.split(',')[1] == 'True'),
```

Comment:
nit:Looks like you might have pyformated using the google3 rules instead of pep8, hence short line lengths.